docs: Add information about the v2 whitelist (#73)
mulesoft/docs-access-management: modules/ROOT/pages/connected-apps-org-admin.adoc

include::_attributes.adoc[]
endif::[]

> As an organization administrator, you are the only user who can view and manage connected apps in Access Management. You can use authorization policies to dictate which apps can be authorized to access user data, and you can whitelist apps that users would like to use.

Reviewer: ```suggestion
As an organization administrator, you are the only user who can view and manage connected apps in Access Management. You can use authorization policies to dictate which apps can be authorized to access user data, and you can whitelist apps that users want to use.
```

To manage your connected apps, navigate to *Access Management* and select the *Connected Apps* tab. You can add apps in the *Owned Apps* section and manage app access in the *Authorizations* section.

> More information about the *Owned Apps* section can be found in the documentation for xref:connected-apps-developers.adoc[app developers].

Reviewer: ```suggestion
For more information about the *Owned Apps* section, see the documentation for xref:connected-apps-developers.adoc[app developers].
```

== Authorizations

> You can use this tab to manage which apps are authorized to access your users' data. Click the *...* icon next to an application to view more details, or to remove the app from your organization.

Reviewer: ```suggestion
You can use the *Authorizations* section to manage which apps are authorized to access your users' data. Click the *...* icon next to an application to view more details or to remove the app from your organization.
```

* Restricted

  Only connected apps that are whitelisted by an administrator can be authorized by organization members to act on their behalf.

> By default, the **Unrestricted** policy is enabled. If you change the policy to **Restricted**, the application whitelist will be enabled and all existing application authorizations will be added automatically.

Reviewer: ```suggestion
By default, the **Unrestricted** policy is enabled. If you change the policy to **Restricted**, the application whitelist is enabled, and all existing application authorizations are added automatically.
```

> == Application Whitelist

Reviewer: ```suggestion
[[application-whitelist-admin]]
== Application Whitelist
```

> When the whitelist is enabled, only whitelisted applications can be used by organization users. If a user attempts to authorize a non-whitelisted app, they will be prompted to contact their organization administrator.

Reviewer: ```suggestion
When the whitelist is enabled, organization users can use only whitelisted applications. If a user attempts to authorize a non-whitelisted app, they are prompted to contact their organization administrator.
```

> image::connected-apps-whitelist-blocked.png[The interface the user sees when an app is not on the whitelist, 40%,40%]

Reviewer: ```suggestion
image::connected-apps-whitelist-blocked.png[The interface the user sees when an app is not on the whitelist, 50%,50%]
```

> Organization administrators will be able to add the app to the whitelist by following the link given to them by the end user.

Reviewer: ```suggestion
Organization administrators are able to add the app to the whitelist by following the link given to them by the end user.
```

> image::connected-apps-add-to-whitelist.png[The interface the admin sees when they whitelist an app, 40%,40%]

Reviewer: ```suggestion
image::connected-apps-add-to-whitelist.png[The interface the administrator sees when they whitelist an app, 50%,50%]
```

> Sometimes, apps will start to ask for additional scopes. This happens when app developers add new features to their products, and need to access more data in your organization.

Reviewer: ```suggestion
Sometimes, apps start to ask for additional scopes. This happens when app developers add new features to their products and need to access more data in your organization.
```

> Previously whitelisted apps _must_ be whitelisted again with the new set of scopes before your users may grant those additional scopes to the user.

Reviewer: ```suggestion
Previously whitelisted apps _must_ be whitelisted again with the new set of scopes before your users can grant those additional scopes to the user.
```

Reviewer: Additional scopes to the app?

> Previously whitelisted apps _must_ be whitelisted again with the new set of scopes before your users may grant those additional scopes to the user.

Author: ```suggestion
Previously whitelisted apps _must_ be whitelisted again with the new set of scopes before your users can grant those additional scopes to the app.
```

> image::connected-apps-add-to-whitelist-again.png[The interface the admin sees when they whitelist an app again, 40%,40%]

Reviewer: ```suggestion
image::connected-apps-add-to-whitelist-again.png[The interface the administrator sees when they whitelist an app again, 50%,50%]
```

> Existing authorizations will continue to work if the app is not whitelisted again. New authorizations using the already-whitelisted set of scopes will also be allowed.

Reviewer: ```suggestion
Existing authorizations continue to work if the app is not whitelisted again. New authorizations using the already-whitelisted set of scopes are also allowed.
```

> Apps that remove scopes will not need to be whitelisted again.

Reviewer: ```suggestion
Apps that remove scopes do not need to be whitelisted again.
```

[NOTE]
--

> Applications created by users in the same organization are marked as *internal apps*, and do not need to be whitelisted manually.

Reviewer: ```suggestion
Applications created by users in the same organization are marked as *internal apps* and do not need to be whitelisted manually.
```
